src/main/java/seedu/address/logic/commands/DeleteSellerCommand.java

```diff
@@ -45,7 +44,7 @@ public CommandResult execute(Model model) throws CommandException {
         requireNonNull(model);
         // Search for the person with the specified phone number
         Client personToDelete = model.getFilteredClientList().stream()
-                .filter(person -> person instanceof Seller)
+                .filter(Client::isSeller)
```

Comment:
same for this class, try to maintain the abstraction by using the Phone class to encapsulate the phoneNumber rather than using the String type